### ISSUE TITLE
Add option to generate different genre of jokes

### ISSUE THREAD

**Issue by ExpressHermes** [ExpressHermes/JokesApp#4]
Show different genres of jokes(Dad jokes/Knock Knock Jokes/Chuck Norris jokes) and use different APIs available to get these jokes when the user clicks one of the genres.

Try to use the APIs that are free and have a short response time. 

There are many APIs that are optimal for our use case. I am providing one of the collections. Feel free to suggest more.
[Sample Collection](https://rapidapi.com/collection/jokes)

**Comment by kartikeyjangir** [comment]
Hey, I would like to work on this. Could you please assign it to me?

**Comment by ExpressHermes** [comment]
@kartikeyjangir I have opened this issue for KWoC 2021 contributors. If you are participating in it, you are welcome to contribute to this issue.

**Comment by kartikeyjangir** [comment]
@ExpressHermes yes, I am participating in KWOC, but assigning an issue is Mandatory or not?

**Comment by ExpressHermes** [comment]
No, assigning issues is not mandatory.

**Comment by KryptonANSHU** [comment]
Hey, I would like to work on this issue
Can I Start working to Contribute?
@ExpressHermes

**Comment by ExpressHermes** [comment]
@KryptonANSHU Yes, make sure to check out which genres have already been implemented to avoid working on duplicate genres.

**Comment by KryptonANSHU** [comment]
> @KryptonANSHU Yes, make sure to check out which genres have already been implemented to avoid working on duplicate genres.

How can I fetch the Code in my local machine of this particular Issue?
I have seen the Changes but How to get em so that I can work on top of them

**Comment by ExpressHermes** [comment]
Check this out: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally

**Comment by Vivek79781** [comment]
is there any genres left??

**Comment by ExpressHermes** [comment]
@Vivek79781 We have only two genres right now. You can add more. There is no fixed list of genres. You can add whatever you think makes a great addition to the app

**Comment by Vivek79781** [comment]
can i use npm packages??

**Comment by ExpressHermes** [comment]
sure

**Comment by copyNinja1219** [comment]
#21  added 3 new genre of jokes . @ExpressHermes
